src/wp-admin/includes/upgrade.php

```diff
@@ -3117,10 +3122,18 @@ function make_site_theme_from_oldschool( $theme_name, $template ) {
 
 		chmod( "$site_dir/$newfile", 0777 );
 
+		$newfile_content = file( "$site_dir/$newfile" );
+		if ( false === $newfile_content ) {
+			return false;
+		}
+
 		// Update the blog header include in each file.
-		$lines = explode( "\n", implode( '', file( "$site_dir/$newfile" ) ) );
+		$lines = explode( "\n", implode( '', $newfile_content ) );
 		if ( $lines ) {
 			$f = fopen( "$site_dir/$newfile", 'w' );
+			if ( ! is_resource( $f ) ) {
```

Comment:
@xknown for consistency:

```php
if ( false === $f ) {
```

as `fopen` returns `false` when it fails to open the file.